### PR TITLE
add worlddreamer_2026_06_20

### PR DESCRIPTION
**Pull request by worldAgents-c**
## Submission checklist

- [x] I added exactly one JSON file for this submission
- [x] I placed the file in the `submissions/` directory
- [x] My JSON follows the submission template in the README
- [x] I used `"N/A"` for any field that is unavailable or not applicable
- [x] I did not modify existing submission files unrelated to this PR

## Submission details

This PR adds the WorldMultiDreamer submission for the RoboCasa365 leaderboard.

The model was evaluated on the RoboCasa365 `target50` setting with `split=target`, `n_envs=5`, `n_episodes=50`, and `n_action_steps=8`.

Evaluation details:

- Model name: Worlddreamer,
- Policy family: Worlddreamer,
- Submission source": "external",
- Robocasa version": "1.0.1",
- Atomic-Seen success": 58.9,
- Composite-Seen success": 18.5,
- Composite-Unseen success": 7.1,
- Code URL": "https://github.com/worldAgents-c/world_dreamer_server-robocasa365-multi_task/tree/main",
- Commit hash": "f9ba61f0657ca79b864e3eac6ddd6ef3634b4645",
- Checkpoint URL": "https://huggingface.co/WorldAgents-c/world_dreamer-robocasa365-multi_task",

Due to environment and access restrictions, we are not able to provide a public W&B run for this submission. We monitored training with local logs and include the training curve below for reference.

Training was run on 10 A100 GPUs with per-GPU batch size 2 and gradient accumulation 2.

Thank you to the RoboCasa365 team for maintaining the benchmark and for supporting community submissions.

### CODE REVIEW

**Comment by sepnasiriany** [comment]
Thank you for submitting! Please give me until the end of the week to verify results

**Comment by GostInShell** [comment]
@sepnasiriany Hi! I noticed that this submission uses `split=target` for evaluation, whereas the [documentation](https://robocasa.ai/docs/build/html/benchmarking/multitask_learning.html) specifies `split=pretrain`. Could you please clarify which split is the correct one to use?
image

**Comment by sepnasiriany** [comment]
@GostInShell yes you're absolutely right, pretrain is the correct split. This passed my eye so I will redo evaluations and update the results based on what I get

**Comment by sepnasiriany** [comment]
Evaluations on pretrain split done, thanks again to @GostInShell for catching this. As expected, the seen splits do better (since those scenes are seen in training) and composite_unseen does a little bit worse. These updated results will be used:

```
[atomic_seen]
CloseBlenderLid    13.33%
CloseFridge    83.33%
CloseToasterOvenDoor    90.00%
CoffeeSetupMug    36.67%
NavigateKitchen    80.00%
OpenCabinet    70.00%
OpenDrawer    70.00%
OpenStandMixerHead    83.33%
PickPlaceCounterToCabinet    80.00%
PickPlaceCounterToStove    70.00%
PickPlaceDrawerToCounter    63.33%
PickPlaceSinkToCounter    90.00%
PickPlaceToasterToCounter    80.00%
SlideDishwasherRack    70.00%
TurnOffStove    26.67%
TurnOnElectricKettle    83.33%
TurnOnMicrowave    46.67%
TurnOnSinkFaucet    56.67%
AGGREGATE    66.30% (358/540)

[composite_seen]
DeliverStraw    13.33%
GetToastedBread    3.33%
KettleBoiling    23.33%
LoadDishwasher    56.67%
PackIdenticalLunches    10.00%
PreSoakPan    46.67%
PrepareCoffee    3.33%
RinseSinkBasin    46.67%
ScrubCuttingBoard    73.33%
SearingMeat    0.00%
SetUpCuttingStation    20.00%
StackBowlsCabinet    56.67%
SteamInMicrowave    13.33%
StirVegetables    23.33%
StoreLeftoversInBowl    3.33%
WashLettuce    33.33%
AGGREGATE    26.67% (128/480)

[composite_unseen]
ArrangeBreadBasket    13.33%
ArrangeTea    0.00%
BreadSelection    13.33%
CategorizeCondiments    0.00%
CuttingToolSelection    10.00%
GarnishPancake    43.33%
GatherTableware    0.00%
HeatKebabSandwich    0.00%
MakeIceLemonade    26.67%
PanTransfer    0.00%
PortionHotDogs    13.33%
RecycleBottlesByType    3.33%
SeparateFreezerRack    0.00%
WaffleReheat    6.67%
WashFruitColander    6.67%
WeighIngredients    6.67%
AGGREGATE    8.96% (43/480)
```

**Comment by worldAgents-c** [comment]
Hi, thanks for maintaining the RoboCasa Leaderboard!  @sepnasiriany 

Our team had not noticed this issue earlier, and we only discovered it recently while preparing a new model for another round of evaluation.

Could you please help us update our leaderboard entry with the following changes?

* Remove the currently cited paper, as it is not associated with our team.
* Change the submitter name to World Agents.
* Replace the current logo with our team logo attached below.
worldAgents_logo

Thanks for your help!

**Comment by sepnasiriany** [comment]
@worldAgents-c thank you for the correction, do you have a paper from your team you'd like me to include?

**Comment by worldAgents-c** [comment]
Thanks! We're currently preparing the paper and plan to release it on arXiv once it's ready.

> @worldAgents-c thank you for the correction, do you have a paper from your team you'd like me to include?
